package.json

```diff
@@ -1,12 +1,12 @@
 {
-  "repository": {
-  },
+  "repository": {},
   "dependencies": {
-    "brunch": "^1.8.5",
     "babel-brunch": "^5.1.1",
+    "brunch": "^1.8.5",
     "clean-css-brunch": ">= 1.0 < 1.8",
     "css-brunch": ">= 1.0 < 1.8",
     "javascript-brunch": ">= 1.0 < 1.8",
+    "sass-brunch": "^1.9.1",
```

Comment:
Though I'm not an expert in frontend technologies, I'm wondering if Sass usage will make it harder to maintain for people with no Sass experience.
Also I've seen there is https://github.com/postcss/postcss available, which seems to provide Sass features but doesn't introduce new syntax.
It's the thing behind well-known https://github.com/postcss/autoprefixer.
:bowtie: